deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/KerasSequentialModel.java

```diff
@@ -29,6 +29,7 @@
 import org.deeplearning4j.nn.modelimport.keras.exceptions.InvalidKerasConfigurationException;
 import org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException;
 import org.deeplearning4j.nn.modelimport.keras.layers.KerasInput;
+import org.deeplearning4j.nn.modelimport.keras.preprocessors.ReshapePreprocessor;
```

Comment:
Is that import here needed? Given the other changes, it looks like it isn't.

Reply:
This was originally added I think because it was the problematic aspect of this when dealing with the channel ordering propagation to different layers.  I updated the PR.